test/integration/embedded_pulsar_job_conf.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<!-- A sample job config that explicitly configures job running the way it is configured by default (if there is no explicit config). -->
```

Comment:
This configures the `pulsar_embed` job runner, right?
Sounds a bit misleading, I  guess you could remove the comment?